src/main/java/org/terasology/launcher/util/FileUtils.java

```diff
@@ -99,7 +99,7 @@ public static boolean extractZipTo(File archive, File outputLocation) {
             try (ZipInputStream zis = new ZipInputStream(new FileInputStream(archive))) {
                 ZipEntry ze;
                 while ((ze = zis.getNextEntry()) != null) {
-                    File extractedFile = new File(outputLocation, com.google.common.io.Files.getNameWithoutExtension(ze.getName()));
+                    File extractedFile = new File(outputLocation, ze.getName());
```

Comment:
If you take a look at https://github.com/MovingBlocks/TerasologyLauncher/issues/400#issuecomment-296599137 it was actually a mistake my part which I didn't spot. The issue was with my test code and not with the `getName()` method. 😓